providers/faux/faux.go

```diff
@@ -43,9 +44,10 @@ func (p *Provider) BeginAuth(state string) (goth.Session, error) {
 func (p *Provider) FetchUser(session goth.Session) (goth.User, error) {
 	sess := session.(*Session)
 	return goth.User{
-		UserID:          sess.ID,
-		Name:            sess.Name,
-		Email:           sess.Email,
+		UserID:   sess.ID,
+		Name:     sess.Name,
+		Email:    sess.Email,
+		Provider: sess.Provider,
```

Comment:
This line is not strictly needed any longer with the simplified method calls. I still think it is better to keep it to be consistent in the test provider.